StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Elements/InlineSignup/LinkInlineSignupView.swift

```diff
@@ -131,16 +156,17 @@ final class LinkInlineSignupView: UIView {
             elements.append(contentsOf: [checkboxElement, combinedEmailNameSection])
         case .checkboxWithDefaultOptIn:
             elements.append(contentsOf: [checkboxElement, defaultOptInElement, combinedEmailNameSection].compactMap { $0 })
-        case .textFieldsOnlyEmailFirst:
-            elements.append(contentsOf: [emailSection, phoneNumberSection, nameSection])
-        case .textFieldsOnlyPhoneFirst:
-            elements.append(contentsOf: [phoneNumberSection, emailSection, nameSection])
+        case .textFieldsOnlyEmailFirst, .textFieldsOnlyPhoneFirst:
+            elements.append(combinedEmailNameSection)
         case .signupOptIn:
             elements.append(checkboxElement)
         }
 
-        let style: FormElement.Style = viewModel.showCheckbox ? .plain : .bordered
```

Comment:
Due to using `combinedEmailNameSection` for all modes, applying the `bordered` style isn't necessary anymore.